custom_components/tuya_local/devices/ips_pro_heatpump.yaml

```diff
@@ -46,13 +46,27 @@ primary_entity:
       name: hvac_action
       type: string
       mapping:
+        - dps_val: smart
```

Comment:
It is incorrect to map this to hvac_action, as it is the hvac_mode, and has no indication of when the device is idle (ie set to warm/cold/smart, but not actually heating/cooling at this time).

I guess the original author of this config did not seem to be able to get hvac_mode working using this, so instead mapped it to the read-only hvac_action instead, but it should be one or the other.

Reply:
would it be possible to map it to the power level? 
is there a way to check the dps_val for >0 ?

but i would need to map the hvac_mode and the power level to be able to determine if it's heating or cooling.